satpy/composites/__init__.py

```diff
@@ -1555,3 +1555,34 @@ def __call__(self, projectables, nonprojectables=None, **info):
 
         masked_projectable = projectable.where(lon_min_max)
         return super(LongitudeMaskingCompositor, self).__call__([masked_projectable], **info)
+
+class MSGQuicklook(GenericCompositor):
+    """Pseudo true color composite.
+        source: https://www.frontiersin.org/articles/10.3389/frsen.2021.666516/full
+
+    Args:
+        ch16_w (float): weight for red channel (1.6 um).
+        ch08_w (float): weight for green channel (0.8 um).
+        ch06_w (float): weight for blue channel (0.6 um).
```

Comment:
These arguments are not used. To use them, an `__init__()` method would be needed. And if so, make them kwargs and have the MSG values as defaults.

Reply:
I'm basing myself on Simon Proud NaturalEnh